pkg/config/datafileprojectconfig/entities/entities.go

```diff
@@ -106,9 +106,9 @@ type Rollout struct {
 
 // Integration represents a integration from the Optimizely datafile
 type Integration struct {
-	Key       string `json:"key"`
-	Host      string `json:"host"`
-	PublicKey string `json:"publicKey"`
+	Key       *string `json:"key"`
```

Comment:
take it as `type`

Reply:
do you mean we should rename it to type? pointer to the `string` is required here since `string` type in golang cannot be nil by itself.

Reply:
discussed offline.